components/org.wso2.carbon.identity.handler.event.account.lock/pom.xml

```diff
@@ -61,9 +61,24 @@
                         </Export-Package>
                         <Import-Package>
                             org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
-                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}"
+                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
+                            org.apache.commons.lang.*; version="${commons-lang.wso2.osgi.version.range}",
+                            org.apache.commons.logging; version="${commons-logging.osgi.version.range}",
+                            org.wso2.carbon.user.api; version="${carbon.user.api.imp.pkg.version.range}",
+                            org.wso2.carbon.user.core.*; version="${carbon.kernel.version}",
+                            org.wso2.carbon.identity.application.common.*; version="${project.version}",
+                            org.wso2.carbon.identity.base; version="${project.version}",
+                            org.wso2.carbon.identity.core.bean.context; version="${project.version}",
+                            org.wso2.carbon.identity.core.handler; version="${project.version}",
+                            org.wso2.carbon.identity.core.util; version="${project.version}",
+                            org.wso2.carbon.identity.event; version="${project.version}",
+                            org.wso2.carbon.identity.event.bean; version="${project.version}",
+                            org.wso2.carbon.identity.event.event; version="${project.version}",
+                            org.wso2.carbon.identity.event.handler; version="${project.version}",
+                            org.wso2.carbon.identity.event.services; version="${project.version}",
+                            org.wso2.carbon.identity.governance; version="${project.version}",
```

Comment:
should be
`org.wso2.carbon.identity.governance;
                            version="${identity.governance.imp.pkg.version.range}",`